README.md

```diff
@@ -0,0 +1,56 @@
+# infinispan-operator
```

Comment:
Missing build instructions. Minimally we need ```dep ensure``` to get dependencies. Is it possible to compile without the operator-sdk, using ```go build```?

Reply:
This will come with https://issues.jboss.org/browse/ISPN-9891?filter=-1